cargo-dist/src/config.rs

```diff
@@ -245,6 +269,8 @@ impl DistMetadata {
             ci,
             installers,
             targets,
+            features,
+            default_features: _,
```

Comment:
default_features would want to be merged

Reply:
Good point! I guess this got lost in too little sleep :)